var/spack/repos/builtin/packages/epics-base/package.py

```diff
@@ -23,6 +23,10 @@ class EpicsBase(MakefilePackage):
     depends_on("readline")
     depends_on("perl", type=("build", "run"))
 
+    def patch(self):
+        filter_file(r"^\s*CC\s*=.*", "CC = " + self.compiler.cc, "configure/CONFIG.gnuCommon")
+        filter_file(r"^\s*CCC\s*=.*", "CCC = " + self.compiler.cxx, "configure/CONFIG.gnuCommon")
```

Comment:
Note that `self.compiler.cxx` is the actual compiler binary *underneath* the Spack compiler wrapper. If you want this to be the spack wrapper you can set `"CC = " + spack_cc` and `... = spack_cxx`

Reply:
(these variables are automatically defined for you and globally available to all methods in a package file)

Reply:
Thanks for the tip! I saw both in https://spack.readthedocs.io/en/latest/packaging_guide.html#filtering-functions so was a little confused.